plugins/javadoc/src/main/kotlin/org/jetbrains/dokka/javadoc/renderer/KorteJavadocRenderer.kt

```diff
@@ -114,8 +114,6 @@ class KorteJavadocRenderer(val context: DokkaContext, resourceDir: String) :
     private fun getTemplateConfig() = TemplateConfig().also { config ->
         listOf(
             TeFunction("curDate") { LocalDate.now() },
-            TeFunction("jQueryVersion") { "3.1" },
-            TeFunction("jQueryMigrateVersion") { "1.2.1" },
```

Comment:
Not clear why these lines were removed, any comments? :)

Reply:
They were removed because they weren't actually being used anywhere and I found it to be misleading. I can add them back in with the appropriate versions. :)